portal/src/http_status_codes.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Purpose: Contains the enumeration of HTTP status codes.
+export enum HttpStatusCodes {
+    OK = 200,
+    CREATED = 201,
+    NO_CONTENT = 204,
+    MOVED_PERMANENTLY = 301,
+    FOUND = 302,
+    TOO_MANY_REDIRECTS = 310,
+    BAD_REQUEST = 400,
+    UNAUTHORIZED = 401,
+    FORBIDDEN = 403,
+    NOT_FOUND = 404,
+    INTERNAL_SERVER_ERROR = 500,
+    LOOP_DETECTED = 508
+}
```

Comment:
Suggestion: I like this way of presenting status codes. However, we are only using (afaics) `LOOP_DETECTED, TOO_MANY_REDIRECTS, NOT_FOUND`. I would propose to adhere to @mlegner 's YAGNI principle, and remove  the others until we need them.

```suggestion
export enum HttpStatusCodes {
    TOO_MANY_REDIRECTS = 310,
    NOT_FOUND = 404,
    LOOP_DETECTED = 508
}
```

Reply:
Fair! 🙆‍♂️